test/manual/amp-video-multiple-selectors.amp.html

```diff
@@ -0,0 +1,337 @@
+      width: 100%;
+    }
+
+    .dropdown-container select {
+      margin: 0 0 20px;
+    }
+
+    .main {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .main,
+    h1,
+    .wrap {
+      max-width: 960px;
+      margin-left: auto;
+      margin-right: auto;
+    }
+
+    .video-container {
+      flex: 1;
+    }
+
+    h1 {
+      font-weight: bold;
+      margin: 40px auto 20px;
+      font-size: 24px;
+    }
+
+    h2 {
+      font-weight: bold;
+      margin: 40px 0 20px;
+      font-size: 14px;
+      display: block;
+      flex: 0 0 100%;
+    }
+
+    aside {
+      margin-left: 20px;
+      min-width: 240px;
+    }
+
+    .analytics-events-container {
+      padding: 10px 0 20px;
+      background: #fafafa;
+      overflow: auto;
+      flex: 1;
+    }
+
+    td {
+      padding: 6px 10px 0 0;
+    }
+
+    .empty>td {
+      padding-top: 0;
+    }
+
+    button {
+      margin-right: 10px;
+    }
+  </style>
+  <style amp-boilerplate>
+    body {
+      -webkit-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      -moz-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      -ms-animation: -amp-start 8s steps(1, end) 0s 1 normal both;
+      animation: -amp-start 8s steps(1, end) 0s 1 normal both
+    }
+
+    @-webkit-keyframes -amp-start {
+      from {
+        visibility: hidden
+      }
+
+      to {
+        visibility: visible
+      }
+    }
+
+    @-moz-keyframes -amp-start {
+      from {
+        visibility: hidden
+      }
+
+      to {
+        visibility: visible
+      }
+    }
+
+    @-ms-keyframes -amp-start {
+      from {
+        visibility: hidden
+      }
+
+      to {
+        visibility: visible
+      }
+    }
+
+    @-o-keyframes -amp-start {
+      from {
+        visibility: hidden
+      }
+
+      to {
+        visibility: visible
+      }
+    }
+
+    @keyframes -amp-start {
+      from {
+        visibility: hidden
+      }
+
+      to {
+        visibility: visible
+      }
+    }
+  </style><noscript>
+    <style amp-boilerplate>
+      body {
+        -webkit-animation: none;
+        -moz-animation: none;
+        -ms-animation: none;
+        animation: none
+      }
+    </style>
+  </noscript>
+  <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+
+<body>
+
+  <div style="display: flex; flex-direction: column; height: 100vh">
+    <div>
+      <h1>amp-video: multiple analytics selectors</h1>
+      <div class="main">
+        <div class="video-container">
+          <amp-video data-vars-video-id="my-video" data-substitutable data-removable-attrs="src" src="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/ForBiggerJoyrides.mp4" width="358" height="204" layout="responsive" id="my-video" controls>
```

Comment:
Sanity check, if we remove the `data-vars-*` for both videos, will it still work?

Reply:
Nope. Because the selector is an array getElements() parses the elements which ultimately end up at getDataVarsElements_(). 
If we want to just use getElement() instead, it would mean going back to something similar to the earlier version of the code or changing getElements().

Reply:
Hmm.. why are the unit tests passing then? I didn't see that any of the fake video elements you created has the `data-vars-*` attribute.

Reply:
I set the data-vars-video-id on lines 50 and 54 and modified the getElements spy to check for getDataVarsElements_().

Reply:
Oh ok, good to know that our tests would've caught this. Since we don't need the video selector to have the data-vars-* restriction, let's change `getElements()` -> `getElementsByQuerySelectorAll_()`, to only do the data-vars-* check conditionally.

Something like:

```
this.root.getElements(context, selectors, selectionMethod, false /** new param*/);


getElements(context, selectors, selectionMethod, useDataVars = true) {
  ...
  return getElementsByQuerySelectorAll_(selectors, useDataVars)
  ...
}


getElementsByQuerySelectorAll_(selectors, useDataVars) {
  ...
  elementArray = useDataVars ? this.getDataVarsElements_(elementArray, selector) : elementArray;
  ...
}
```